ci/playbooks/deploy_watcher_service.yaml

```diff
@@ -28,6 +28,16 @@
               oc wait OpenStackControlPlane controlplane --namespace=openstack
               --for=condition=Ready --timeout=10m
 
+        - name: Patch prometheuses.monitoring.rhobs metric-storage to enable Admin API
+          ansible.builtin.command:
+            cmd: >-
+              oc patch prometheuses.monitoring.rhobs metric-storage --namespace=openstack --type=merge -p '{"spec":{"enableAdminAPI":true}}'
```

Comment:
we can add `enableAdminAPI: true` in https://github.com/openstack-k8s-operators/watcher-operator/blob/main/ci/files/configure_telemetry_for_watcher_service.yaml#L18 . It will take care of that while enabling prometheus.